src/Raven.Server/Documents/Indexes/Static/StaticIndexBase.cs

```diff
@@ -93,36 +93,6 @@ public dynamic LoadDocument(object keyOrEnumerable, string collectionName)
                 keyOrEnumerable.GetType().FullName + ": " + keyOrEnumerable);
         }
 
-        protected IEnumerable<AbstractField> CreateField(string name, object value, bool stored = false, bool? analyzed = null)
```

Comment:
This cannot be deleted. It can be used by user for creating custom fields e.g. here https://github.com/ravendb/ravendb/blob/v4.0/test/SlowTests/Issues/RavenDB_3375.cs#L24